### PR TITLE
Add scripts and adjust plumbing for many temp items

### PR DESCRIPTION
**Pull request by neuromancerxi**
Adds Scripts for items.
Adds Effect scripts for X_BOOST_II
Updates Effect scripts for the following:

ACCURACY_BOOST
ATTACK_BOOST
INTENSION
MAGIC_SHIELD
MULTI_STRIKES
MULTI_SHOTS
PAX
PHYSICAL_SHIELD
POTENCY
RAMPART

Adjusts item_usable use times to 1s.

Notes on effects:

Ascetics Tonic/Gambir - +25/+50 MATT/MACC
A big Thank You to Nyu for confirming the Intension effect for MACC.
https://www.bg-wiki.com/bg/Ascetic's_Tonic
https://www.bg-wiki.com/bg/Ascetic's_Gambir

Bravers Drink - +15 to All Stats
https://www.bg-wiki.com/bg/Braver's%20Drink
Reference to Icons/Effects - https://youtu.be/JYT5a_pTA3o?t=20

Champions Tonic - +15 Haste / +3 Crit rate
Champions Drink/Gambir - +18 Haste / +5% Crit rate
Expected Haste effect to be Magic pool based on BG comment (18 and 15)
Critical Hit rate, no data available from community sources, conservative value of 5 (Drink/Gambir) 3 (Tonic)
Both BG and 'clopedia sources confirm Critical Hit Rate (as does the effect description), however, only BG has a reference to Haste value.
https://www.bg-wiki.com/bg/Champion's_Tonic

Gnostics Drink - Enmity -10
No community resources confirm this value, using SCH Animus Minuo as an reference.
https://www.bg-wiki.com/bg/Gnostic's%20Drink
https://www.bg-wiki.com/bg/Animus_Minuo

Monarchs Drink - Regain +3 (30/1000 per tick) for 3 minutes.
https://www.bg-wiki.com/bg/Monarch's_Drink

Stalwart's Tonic/Gambir - ACC/RACC 50/100 ATTP/RATTP 25/50
A big Thank You to Nyu for confirming the effects apply ACC/RACC and ATTP/RATTP across two effects.
https://www.bg-wiki.com/bg/Stalwart's_Tonic
https://www.bg-wiki.com/bg/Stalwart's_Gambir

Berserker's Tonic/Drink - DA 50/100
Thanks to Nyu for confirming the MULTI_STRIKES effect and 1m duration.
https://ffxiclopedia.fandom.com/wiki/Berserker%27s_Drink
No full data on DA rate, minus 'clopedia which has verification tags. Working on the expectation that this follows a good portion of the other items and follows the half potency values for the lesser item.

Swiftshot Tonic/Drink - Double Shot 50/100
Reference: https://www.ffxiah.com/forum/topic/28603/fools-tonic-broken#1749569

Dusty Scroll of Reraise - Reraise III, 10m duration.
https://www.bg-wiki.com/bg/Dusty_Reraise

Spiritual Incense/Fools Drink/Tonic/Powder: See effects/magic_shield
Carnal Incense/Fanatics Drink/Tonic/Powder: See effects/physical_shield

**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

**Review by Xaver-DaRed — DISMISSED** [review]
LGTM

File: scripts/globals/items/bottle_of_berserkers_drink.lua [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    local power = 100 --DA
+    local duration = 60
+
+    if target:hasStatusEffect(ename) then
+        local buff = target:getStatusEffect(ename)
+        local effectpower = buff:getPower()
+        if effectpower > power then
+            target:messageBasic(xi.msg.basic.NO_EFFECT)
+        else
+            target:addStatusEffect(ename, power, 0, duration, 0, subp)
```
Comment (Xaver-DaRed) [inline scripts/globals/items/bottle_of_berserkers_drink.lua]: subp undefined

File: scripts/globals/items/bottle_of_berserkers_tonic.lua [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    local power = 50 --DA
+    local duration = 60
+
+    if target:hasStatusEffect(ename) then
+        local buff = target:getStatusEffect(ename)
+        local effectpower = buff:getPower()
+        if effectpower > power then
+            target:messageBasic(xi.msg.basic.NO_EFFECT)
+        else
+            target:addStatusEffect(ename, power, 0, duration, 0, subp)
```
Comment (Xaver-DaRed) [inline scripts/globals/items/bottle_of_berserkers_tonic.lua]: Same here and below. subp undefined

File: scripts/globals/items/bottle_of_berserkers_drink.lua (hunk 1 above)
Comment (neuromancerxi) [reply scripts/globals/items/bottle_of_berserkers_drink.lua]: Copy/Paste typo. Fixed.

File: scripts/globals/items/bottle_of_berserkers_tonic.lua (hunk 2 above)
Comment (neuromancerxi) [reply scripts/globals/items/bottle_of_berserkers_tonic.lua]: Copy/Paste typo. Fixed.

File: scripts/globals/items/pinch_of_fanatics_powder.lua [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    local duration = 60
+    local power = 1
+    local wsmitigation = 1
+
+    local function addShield(target, power, duration)
+        target:delStatusEffect(xi.effect.MAGIC_SHIELD)
+        target:addStatusEffect(shieldtype, power, 0, duration, 0, wsmitigation)
+        target:messageBasic(xi.msg.basic.GAINS_EFFECT_OF_STATUS, shieldtype)
+    end
+
```
Comment (Xaver-DaRed) [inline scripts/globals/items/pinch_of_fanatics_powder.lua]: whitespace here.
Also, it seems like the CI doesnt like something you are doing with target, power and duration in some scripts.

    scripts/globals/items/pinch_of_fanatics_powder.lua:21:30: (W432) shadowing upvalue argument target on line 15
    scripts/globals/items/pinch_of_fanatics_powder.lua:21:38: (W431) shadowing upvalue power on line 18
    scripts/globals/items/pinch_of_fanatics_powder.lua:21:45: (W431) shadowing upvalue duration on line 17

File: scripts/globals/items/pinch_of_fanatics_powder.lua [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    return 0
+end
+
+item_object.onItemUse = function(target)
+    local shieldtype = xi.effect.PHYSICAL_SHIELD
+    local duration = 60
+    local power = 1
+    local wsmitigation = 1
+
+    local function addShield(target, power, duration)
```
Comment (claywar) [inline scripts/globals/items/pinch_of_fanatics_powder.lua]: target is already defined in the main function.  My recommendation would be to not include this function definition within an existing function, but rather outside of it.

**Comment by neuromancerxi** [comment]
Cleaned up the scripts and moved most functions to item_utils.

**Comment by WinterSolstice8** [comment]
Looks good to me, but we have conflicts now

**Comment by neuromancerxi** [comment]
Conflicts resolved, fixed some namespace issues, and updated PHYS_ABSORB to percent from 10000 scale in `physical_shield` effect.

**Review by WinterSolstice8 — APPROVED** [review]
Looks good to me

**Review by claywar — APPROVED** [review]

**Pull request merged**